### ISSUE TITLE
[Issue] Value/name editing bug for frigates

### ISSUE THREAD

**Issue by GKboki** [goatfungus/NMSSaveEditor#31]
When editing values or names for frigates, I'll have to click on other values after editing this value before clicking on the next frigate, otherwise the value will set back to the original value before editing. To me it looks like the value for one frigate doesn't auto save before moving on to the other frigates, unless clicking on other values in that frigate. Just a minor problem, but really convenient if fixed.

**Comment by goatfungus** [comment]
Yes, this is currently being fixed.... for now, when you make an edit to a field, click in another field afterwards to get it to save.
